client/lib/i18n-utils/utils.js

```diff
@@ -174,3 +174,33 @@ export function getForumUrl( localeSlug = getLocaleSlug() ) {
 	}
 	return `//en.forums.wordpress.com`;
 }
+
+let langRevisions = {};
```

Comment:
Hmm. I'm not a big fan of storing things like this at module level. Among other things, this isn't great for testing (see the required mocks). Furthermore, it doesn't really work for server-side rendering, since it would pollute across requests there (which is why I like to isolate things at request context level) -- BTW this is one of the main reasons why we can't use `i18n-calypso` for server-side rendering yet.

Can we maybe store this somehow at context level, or in Redux state instead?

Reply:
That's a great point. The client side would need some extra refactoring work and could bloat this PR, so I've separated it out to https://github.com/Automattic/wp-calypso/pull/27179/ and keep the server-side bits here.